.gitattributes

```diff
@@ -1,6 +1,7 @@
 /.* export-ignore
 apigen* export-ignore
 CODE_OF_CONDUCT.md export-ignore
+CHANGELOG-Archive.txt export-ignore
```

Comment:
This change is not relevant to the stated purpose of this PR, it should be backed out.